src/components/schemas/Yield.yaml

```
@@ -0,0 +1,10 @@
+type: object
+properties:
+  gross:
+    type: number
+    format: double
+    example: 2530
+  net:
+    type: number
+    format: double
+    example: 2530
```

suggestion: Ajouter un saut de ligne à la fin du fichier.
```suggestion
    example: 2530

```